scopes/html/html/utils.ts

```diff
@@ -0,0 +1,32 @@
+import { HtmlComposition } from './interfaces';
+import { renderTemplate } from '@teambit/html.modules.render-template';
+
+/**
+ * this mounts compositions into the DOM in the component preview.
+ * this function can be overridden through ReactAspect.overrideCompositionsMounter() API
+ * to apply custom logic for component DOM mounting.
+ */
+export const RenderHtmlComposition = (target: HTMLElement | null, composition: HtmlComposition) => {
```

Comment:
change the file name to render-html-composition or render-composition.

Reply:
Done